.github/workflows/python_unit_test.yaml

```diff
@@ -7,6 +7,10 @@ on:
         description: 'A URL identifying a Bash script that can run commands needed for service-specific test setup'
         required: true
         type: string
+      dependency-file-location:
```

Comment:
the dependencies should not be copied here, each repo already has its own dependency file. When we update the dependencies in a branch, we shouldn't be running the tests with another set of dependencies.

So for fence we should install using poetry. I think it would be fair to assume in this reusable workflow that we always use poetry, since we eventually want to update all our repos to use it

Reply:
Exactly, we should be using poetry across the board and especially in these standard worflows

Reply:
@paulineribeyre The dependencies are in this, because I believe there are some dependencies that are installed in Travis, that are not currently captured in the dependencies in the fence repo. I can double check, but when I was running these, those dependencies needed to be there for all the tests to pass. We can definitely cut those dependencies, but I think to get the tests to work, we'd need to take a look at fence, which may be the better way to do it.

Reply:
Then those dependencies should be added to the `dev-dependencies` in the repo

Reply:
Sounds good, I'll take care of that and update once it's complete.